src/js/client.ts

```diff
@@ -15,8 +16,20 @@ import { Alert, LogBox, YellowBox } from 'react-native';
 import { ReactNativeClientOptions } from './options';
 import { NativeTransport } from './transports/native';
 import { createUserFeedbackEnvelope } from './utils/envelope';
+import { SDK_NAME, SDK_PACKAGE_NAME, SDK_VERSION } from './version';
 import { NATIVE } from './wrapper';
 
+const defaultSdkInfo: SdkInfo = {
```

Comment:
The `sdkinfo.ts` does that directly in the event, can we somehow unify this behavior? eg extracting and reusing the method.

Reply:
yes